"No Requirements"

[ERROR] addons/utime-master/lua/autorun/sv_utime.lua:25: attempt to index global 'ULib' (a nil value)

We will get the readme fixed. Thank you!

On Thu, Mar 19, 2020 at 22:47 daoshii <notifications@github.com> wrote:

> [ERROR] addons/utime-master/lua/autorun/sv_utime.lua:25: attempt to index
> global 'ULib' (a nil value)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TeamUlysses/utime/issues/8>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAEBTY2VBLEVZ772IXJ6LLRILKKXANCNFSM4LP5P65A>
> .
>

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ Utime is brought to you by..
 * Brett "Megiddo" Smith - Contact: <megiddo@ulyssesmod.net>
 
 ## Requirements
-Utime has no requirements.
+Utime requires [ULib](https://github.com/TeamUlysses/ulib).
 
 ## Installation
 To install Utime, simply extract the files from the downloaded archive to your garrysmod/addons folder.

